cea/datamanagement/zone_helper.py

```diff
@@ -295,9 +295,9 @@ def flatten_geometries(geometry):
         from shapely.ops import unary_union
         if geometry.type == 'Polygon':  # ignore Polygons
             return geometry
-        elif geometry.type == 'Point':
-            print('Discarding geometry of type: Point')
-            return None # discard geometry if it is a Point
+        elif geometry.type in ['Point', 'LineString']:
+            print('Discarding geometry of type: {}'.format(geometry.type))
```

Comment:
I think we can start using [f-strings](https://docs.python.org/3/reference/lexical_analysis.html#f-strings) (introduced in Python 3) for formatting strings. It looks neater and more concise. 

e.g.
```suggestion
            print(f'Discarding geometry of type: {geometry.type}')
```

Reply:
Updated accordingly!